### PR TITLE
[Docs] Harmonise README with docs.

### LINKED ISSUE #54: Make README consistent with docs
- [x] Add contributors
- [x] Add coverage badge
- [x] Add docs build badge

### PR DESCRIPTION
**Pull request by Roland-djee**
Closes #54

### CODE REVIEW

**Comment by madagra** [comment]
@Roland-djee you can use symlink to the `index.md`. Github supports the rendering differently than Gitlab.

**Comment by nmheim** [comment]
Once we have to docs build set up properly (working on it) we should update the links to `.md` files as well. They are not converted to working docs pages links because the README is not built via mkdocs.

File: README.md [hunk 1]
```diff
@@ -1,74 +1,71 @@
...
+* [High-order generalized parameter shift rules](link to psr tutorial) for _**differentiating parametrized quantum operations**_.
 
-* Out-of-the-box automatic differentiability of quantum programs using [https://pytorch.org](https://pytorch.org)
+* Out-of-the-box _**automatic differentiability**_ of quantum programs with [PyTorch](https://pytorch.org/) integration.
 
-* `QuantumModel`s to make `QuantumCircuit`s differentiable and runnable on a variety of different
-  backends like state vector simulators, tensor network emulators and real devices.
+* _**Efficient execution**_ on a variety of different purpose backends: from state vector simulators to tensor network emulators and real devices.
 
-Documentation can be found here: [https://pasqal-qadence.readthedocs-hosted.com/en/latest](https://pasqal-qadence.readthedocs-hosted.com/en/latest).
```
Comment (awennersteen) [inline README.md:29]: we should already know the public RTD address, right? can't we just put it in in this MR?

Comment (Roland-djee) [reply README.md:29]: Thanks, done.

**Comment by Roland-djee** [comment]
@madagra Could you please have a quick look at this one ?

**Pull request merged**